Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -106,31 +106,13 @@ namespace AZ::SceneAPI::Behaviors
         };
 
         using MeshTransformEntry = AZStd::pair<Containers::SceneGraph::NodeIndex, MeshNodeData>;
```

Comment:
Suggestion: Might be worth renaming MeshTransformEntry to MeshDataEntry.

Reply:
will do!